zipline/finance/performance/tracker.py

```diff
@@ -84,9 +83,7 @@ class PerformanceTracker(object):
     """
     Tracks the performance of the algorithm.
     """
-
-    @with_environment()
-    def __init__(self, sim_params, env=None):
+    def __init__(self, sim_params):
```

Comment:
Why doesn't this use `with_environment` anymore?  Was it just unused?

I notice we're pulling the environment directly off the `trading` module, which is an idiom I'd like to purge.  I'd rather use the `with_environment` decorator to supply it as an argument and use that argument instead of `trading.environment` attribute accesses.